docs/iris/src/whatsnew/2.0.rst

```diff
@@ -192,6 +192,18 @@ Incompatible Changes
 * If the `packing` argument to `iris.save` is a dictionary, an error is raised
   if it contains any keys other than 'dtype', 'scale_factor' and 'add_offset'.
 
+* The deprecated :mod:`iris.fileformats.grib` was removed. All Iris GRIB
+  functionality is now delivered through :mod:`iris-grib`.
+
+* In Iris v1 it was possible to configure Iris to log at import time through
+  :attr:`iris.config.LOGGING`. This capability has been removed in iris v2.
```

Comment:
Does this attribute actually still exist or will we end up with a broken link in the docs?

Reply:
Could you also capitalise the last "iris" to be consistent with the other examples of the word in this point please.

Reply:
FWIW the what's new can be read at https://scitools-docs.github.io/iris/v2.0.x/whatsnew/2.0.html (once the cache is updated, it will also be available at http://scitools.org.uk/iris/docs/v2.0/whatsnew/2.0.html)

Reply:
OK I've checked the built whatsnew and there isn't a link to LOGGING. It doesn't really impact the page though because the text falls back to preformatted.